api/src/main/java/org/pmiops/workbench/api/OfflineRdrExportController.java

```diff
@@ -63,14 +68,15 @@
       try {
         groupIdsAndPushTask(rdrExportService.findAllUserIdsToExport(), EXPORT_RESEARCHER_PATH);
       } catch (Exception ex) {
-        log.severe(
-            String.format("Error while exporting researcher data to RDR: %s", ex.getMessage()));
+        throw new ServerErrorException(
+            String.format("Error creating RDR export Cloud Tasks for users: %s", ex.getMessage()));
       }
       try {
         groupIdsAndPushTask(rdrExportService.findAllWorkspacesIdsToExport(), EXPORT_USER_PATH);
       } catch (Exception ex) {
-        log.severe(
-            String.format("Error while exporting workspace data to RDR: %s", ex.getMessage()));
+        throw new ServerErrorException(
+            String.format(
+                "Error creating RDR export Cloud Tasks for workspaces: %s", ex.getMessage()));
```

Comment:
If you use this [constructor](https://docs.oracle.com/javaee/7/api/javax/ws/rs/ServerErrorException.html#ServerErrorException-javax.ws.rs.core.Response-java.lang.Throwable-) that takes the throwable itself, you keep the stack trace.

Reply:
Thanks – done